src/utils/parsingModel.json

```diff
@@ -2696,5 +2696,201 @@
         }
       ]
     }
+  },
+ "Activité du journal des accès/Activités _ liste des services Google auxquels vos.csv": {
+    "language": "fr",
+    "comment": "list all google services used by the suer",
+    "file_structure_properties": {
+      "header": true,
+      "delimiter": ","
+    },
+    "entries": [
+      {
+        "action_type": "Google services list"
+      },
+      {
+        "data_origin": "Inferred"
+      },
+      {
+        "data_type": "Contractual"
+      },
+      {
+        "platform": "google"
+      },
+      {
+        "timestamp": ""
+      },
+      {
+        "details": ""
+      }
+    ]
+  },
+ "Activité du journal des accès/Appareils _ liste des appareils (par exemple, Nest.csv": {
```

Comment:
De mon côté, j'ai le path suivant en version EN : `Devices – A list of devices (i.e. Nest, Pixel, iPh.csv`.

Ca fonctionne en FR ? Il ne manque pas `,Pixel, iPh.csv` ?

Reply:
non ça fonctionne, je me demande s'ils n'ont pas une limite sur la longueur du nom de fichier et qui tronque

Reply:
idem tu pourras tester l'EN ?